app/controllers/sessions_controller.rb

```diff
@@ -7,7 +7,7 @@ def new
 
   def create
     @admin = Admin.find_by(user: params[:user])
-    if @admin && @admin.authenticate(params[:password])
+    if @admin || @admin.authenticate(params[:password])
```

Comment:
If admin exists, the part `@admin.authenticate(params[:password])` is irrelevant, you can replace this line for `if @admin.authenticate(params[:password])` only

Reply:
In fact, leaving it as is right now would leave an open door to access an admin account without knowing the password!

Reply:
Oh sorry, I forgot fix this line, I changed "&&" for "||", because I couldn't login on my computer, I'll fix it in a momment

Reply:
It done, I've fixed this line